models/gamesM.js

```diff
@@ -66,3 +66,87 @@ exports.addReviewVotes = (reviewID, voteInc) => {
         });
     });
 };
+
+exports.getReviewListComments = (categoryObj) => {
+  validCategories = [
+    "review_id",
+    "owner",
+    "title",
+    "category",
+    "review_img_url",
+    "created_at",
+    "votes",
+    "designer",
+    "comment_count",
+  ];
+  return db
+    .query(
+      `SELECT users.username, COUNT(reviews.owner) FROM reviews 
+  JOIN users
+  ON reviews.owner = users.username
+  GROUP BY users.username;`
+    )
+    .then((countObj) => {
+      countObj = countObj.rows;
+      let countObj2 = {};
+      countObj.forEach((user) => {
+        countObj2[user.username] = user.count;
+      });
+      return Promise.all([
+        db.query(`ALTER TABLE reviews
+      ADD comment_count INT;
+      `),
+        countObj2,
+      ]);
+    })
+    .then((promise) => {
+      let countObj = promise[1];
+      let promiseArr = [];
+      for (owner in countObj) {
+        promiseArr.push(
+          db.query(
+            `UPDATE reviews
+            SET comment_count=$1 
+            WHERE owner=$2;`,
+            [countObj[owner], owner]
+          )
+        );
+      }
+      return Promise.all(promiseArr);
+    })
+    .then(() => {
+      dbRequest = `SELECT * FROM reviews`;
+
```

Comment:
So, although this might work, it doesn't show you understand the benefits of a relational database. 
You have reviews, they are linked to comments via a foreign key. 

You can create a join to link these two. 

Now you have all the comments for each review, you can count them. 

This leaves the original tables an set up in tact, but still enables you to get the data you want. At the moment, you're changing the database which could cause issues in other places/tests. 

You should be able to get the comment count in 5 lines or so, using a join.